CITATION.cff

```diff
@@ -0,0 +1,45 @@
+cff-version: 1.2.0
+title: "GLASS: Generator for Large Scale Structure"
+message: >-
+  If you use this software, please cite the associated paper instead.
+authors:
+  - given-names: Nicolas
+    family-names: Tessore
+    orcid: https://orcid.org/0000-0002-9696-7931
+  - given-names: Arthur
+    family-names: Loureiro
+    orcid: https://orcid.org/0000-0002-4371-0876
+  - given-names: Benjamin
+    family-names: Joachimi
+    orcid: https://orcid.org/0000-0001-7494-1303
+  - given-names: Maximilian
+    family-names: von Wietersheim-Kramsta
+    orcid: https://orcid.org/0000-0003-4986-5091
+  - given-names: Niall
+    family-names: Jeffrey
+    orcid: https://orcid.org/0000-0003-2927-1800
+doi: 10.21105/astro.2302.01942
+repository-code: "https://github.com/glass-dev/glass"
+url: "https://glass.readthedocs.io"
+keywords:
+  - Astrophysics
+  - Cosmology
+  - Nongalactic Astrophysics
+license: MIT
+preferred-citation:
+  authors:
+    - family-names: Nicolas
+      given-names: Tessore
+      orcid: https://orcid.org/0000-0002-9696-7931
```

Comment:
Under [Credit redirection](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#credit-redirection), it states:

> your CITATION.cff should contain metadata about the software at the root of the CITATION.cff file

Therefore, should we not remove `doi` from the root of the file, and move the full `authors` list under the `preferred-citation` key?

Reply:
So ideally we would want to re-open #179, add that stuff at the top, and everything paper specific `preferred-citation`. Akin to https://github.com/astro-informatics/sleplet/blob/main/CITATION.cff

Reply:
I was thinking that we would probably add the DOI if we produce a write-up of the Array API version for, e.g., JOSS or so.

Reply:
So maybe for now we just "abuse" the `CITATION.cff` syntax bit and get rid of `preferred-citation` and just make everything strictly about the GLASS paper?

Reply:
I thought we should put the GLASS paper into `preferred-citation` and simply not add that same info to the root of the file. That is exactly how the example in the spec does it.

Reply:
Okay cool, that was my initial thought ^